example/spec/parallel/lzardsBackup/LzardsBackupSpec.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+const get = require('lodash/get');
+const pAll = require('p-all');
+const path = require('path');
+const pTimeout = require('p-timeout');
+
+const { createCollection } = require('@cumulus/integration-tests/Collections');
+const { deleteCollection } = require('@cumulus/api-client/collections');
+const { lambda } = require('@cumulus/aws-client/services');
+const { putFile } = require('@cumulus/aws-client/S3');
+
+const { loadConfig } = require('../../helpers/testUtils');
+
+describe('The Lzards Backup Task ', () => {
+  let beforeAllFailed = false;
+  let collection;
+  let FunctionName;
+  let functionConfig;
+  let prefix;
+  let provider;
+  let ingestBucket;
+  let ingestPath;
+  let lzardsBackupOutput;
+
+  beforeAll(async () => {
+    try {
+      const config = await loadConfig();
+      prefix = config.stackName;
+      ingestBucket = config.buckets.protected.name;
+      ingestPath = `${ingestBucket}/${prefix}/lzardsBackupSpec`;
+      await putFile(ingestBucket, `${prefix}/lzardsBackupSpec/testGranule.dat`, path.join(__dirname, 'test_data', 'testGranule.dat'));
+      await putFile(ingestBucket, `${prefix}/lzardsBackupSpec/testGranule.jpg`, path.join(__dirname, 'test_data', 'testGranule.jpg'));
+      FunctionName = `${prefix}-LzardsBackup`;
+      functionConfig = await lambda().getFunctionConfiguration({
+        FunctionName,
+      }).promise();
+
+      // Create the collection
+      collection = await createCollection(
+        prefix,
+        {
+          files: [
+            {
+              bucket: 'protected',
+              regex: '^[^.]+\.jpg$',
+              lzards: { backup: true },
+              sampleFileName: 'asdf.jpg',
+            },
+            {
+              bucket: 'protected',
+              regex: '^[^.]+\.dat$',
+              sampleFileName: 'asdf.dat',
+            },
+          ],
+        }
+      );
+
+      const Payload = JSON.stringify({
```

Comment:
Is there a reason why `Payload` is capitalized?

Reply:
Great question! 

Yes: 

lambda.invoke() utilzes `FunctionName` and `Payload` as method arguments.  As such, if this were to be `payload` we'd have to update the lower line to payload: Payload, which is arguably less readable.     I

```
      lzardsBackupOutput = await pTimeout(
        lambda().invoke({ FunctionName, payload: Payload }).promise(),
        (functionConfig.Timeout + 10) * 1000
      );
```

I could see the argument that it could be mistaken for a class, particularly outside of TS, however as it's used immediately afterword 

See: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Lambda.html#invoke-property